app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -19,12 +19,12 @@ class IrsAttemptsApiController < ApplicationController
 
     def create
       if timestamp
-        result = encrypted_security_event_log_result
+        send_data security_event_tokens
 
-        headers['X-Payload-Key'] = Base64.strict_encode64(result.encrypted_key)
-        headers['X-Payload-IV'] = Base64.strict_encode64(result.iv)
-        send_data Base64.strict_encode64(result.encrypted_data),
-                  disposition: "filename=#{result.filename}"
+        # headers['X-Payload-Key'] = Base64.strict_encode64(rejsult.encrypted_key)
+        # headers['X-Payload-IV'] = Base64.strict_encode64(result.iv)
+        # send_data Base64.strict_encode64(result.encrypted_data),
+        #           disposition: "filename=#{result.filename}"
```

Comment:
We will probably still want to set these headers, I suppose?

Reply:
(Also, if restoring these, note I somehow got an errant 'j' in 'result'...)

Reply:
we definitely want the filename, we don't need the `X-Payload-Key`, `X-Payload-IV` since there's no more whole-body encryption

Reply:
I am optimistic, but not completely convinced, that we've reached consensus on that. It certainly _seems_ unnecessary, but it sort of did early on.

Reply:
It was decided these were needed.